pokedex/model/template/csr/write/fcsr_003.asl

```diff
@@ -0,0 +1,6 @@
+let _ok1 = Write_FFLAGS(value);
+assert _ok1.is_ok;
+let _ok2 = Write_FRM(value);
+assert _ok2.is_ok;
```

Comment:
writing to fcsr never fails.

Reply:
It should never fails, this is an internal constraint to each of the FFLAGS and FRM CSR operation implementation should not fail.